_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: OTel trace group
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# OTel trace group
+
+The `OTel trace group` processor completes missing trace group related fields in the collection of [span](https://github.com/opensearch-project/data-prepper/blob/834f28fdf1df6d42a6666e91e6407474b88e7ec6/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend. The `OTel trace group` processor iedntifies the missing trace group information for a `spanId` by looking up the relevant fields in its root `span` stored in OpenSearch, or in the Amazon OpenSearch Service backend that the local Data Prepper host ingests.
+
+## Usage
+
+<!---Other sections introduce a YAML file here.--->
+
+### OpenSearch
+
+<!---Need an introduction here--->
+
+```
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://localhost:9200"]
+        cert: path/to/cert
+        username: YOUR_USERNAME_HERE
+        password: YOUR_PASSWORD_HERE
+```
+
+See [OpenSearch Security](https://github.com/opensearch-project/data-prepper/blob/834f28fdf1df6d42a6666e91e6407474b88e7ec6/data-prepper-plugins/opensearch/opensearch_security.md#L4) for a detailed explanation.
+
+### Amazon OpenSearch Service
+
+<!--- Needs an introduction.--->
+
+```
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://your-amazon-opensearch-service-endpoint"]
+        aws_sigv4: true
+        cert: path/to/cert
+        insecure: false
+```
+
+See [OpenSearch Sink Security](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/security.md) for a detailed explanation <!---...explanation of?--->.
+
+## Configuration
+
+You can configure the Amazon OpenSearch Service with the following options.
+
+| Name | Required | Description | Default value
+| -----| ---- | -----------|
+|`hosts`| Yes | A list of IP addresses of OpenSearch nodes.
+|`cert` | No | A certificate authority (CA) certificate that is PEM-encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that OpenSearch is using. | `null` |
+|`aws_sigv4` | Yes | A boolean flag to sign the HTTP request with AWS credentials. Only applies to Amazon OpenSearch Service. See [security](security.md) for details. | `false`. |
```

Comment:
Need an updated link for this page.

Reply:
It refers to this file: https://github.com/opensearch-project/data-prepper/blob/129524227779ee35a327c27c3098d550d7256df1/data-prepper-plugins/opensearch/security.md